.ci.yaml

```diff
@@ -1379,12 +1379,26 @@ targets:
 
   - name: Linux_android android_defines_test
     recipe: devicelab/devicelab_drone
-    presubmit: false
+    presubmit: true
+    bringup: true
     timeout: 60
+    dimensions: {
+      kvm: "1",
```

Comment:
whats "kvm" do?

Reply:
Kvm is for the nested virutalization. It supports running of the emulator.